test/parallel/test-tls-connect.js

```diff
@@ -37,7 +37,7 @@ const path = require('path');
   const cert = fs.readFileSync(path.join(common.fixturesDir, 'test_cert.pem'));
   const key = fs.readFileSync(path.join(common.fixturesDir, 'test_key.pem'));
 
-  const options = {cert: cert, key: key, port: common.PORT};
+  const options = {cert: cert, key: key, port: 0};
   const conn = tls.connect(options, common.mustNotCall());
 
   conn.on('error', common.mustCall());
```

Comment:
Nit: Now that you are doing this, it would be better if you could assert the actual error messages. On my machine I got

```
{ Error: connect ECONNREFUSED 127.0.0.1:12346
    at Object.exports._errnoException (util.js:1042:11)
    at exports._exceptionWithHostPort (util.js:1065:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1115:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 12346 }
```